sphinx/source/docs/user_guide/interaction/callbacks.rst

```diff
@@ -124,7 +124,7 @@ available event classes using the ``display_event`` function in order to
 generate the ``CustomJS`` objects. This function is used to update the ``Div``
 with the event name (always accessible from the ``event_name``
 attribute) as well as all the other applicable event attributes. The
-result is a plot that when interacted with, displays the corresponding
+result is a plot that, when interacted with, displays the corresponding
```

Comment:
another that-comma cringe, maybe just delete "when interacted with", or move to the end?

Reply:
Absolutely. I have also taken this chance to change that bit from passive to active voice: 
"The result is a plot that displays the corresponding event on the right when the user interacts with it:"